File: js/classes e objetos/Exemplo 3/Retangulo.js
class Retangulo{
    constructor(_base, _altura)
    {
        this.base=_base;
        this.altura=_altura;
    }
    calculaPerimetro()
    {
        return this.base*2+this.altura*2;
    }
    calculaArea()
    {
        return this.base*this.altura;
    }
}
const retangulo1 = new Retangulo(10.0,3.5);
const retangulo2= new Retangulo(8.0,2.0);

const diffPerimetro = retangulo1.calculaPerimetro() - retangulo2.calculaPerimetro();
console.log(`A diferença dos perimetro é ${diffPerimetro}`);
const diffArea = retangulo1.calculaArea() - retangulo2.calculaArea();
console.log(`A diferença das áreas é ${diffArea}`);

File: py/classes e objetos/Exemplo 2/Retangulo.py
class Retangulo:
    
    def __init__(self,altura,base):
        self.Altura = altura
        self.Base = base

   
    def CalculaPerimetro(self):
        return (2*self.Altura)+(2*self.Base)
    def CalculaArea(self):
        return (self.Altura*self.Base)
    
base = float(input("Digite o valor da base "))
altura=float(input("Digite a altura "))
retangulo1 = Retangulo(altura,base)
print(retangulo1.CalculaPerimetro())
print(retangulo1.CalculaArea())
        